### PR TITLE
[Draft]Try to enable Azure 2 GPUs test box working with circleci

### PR DESCRIPTION
**Pull request by gunandrose4u**
Try to enable Azure 2 GPUs test box working with circleci

### CODE REVIEW

File: test/distributed/test_c10d.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 from torch.testing._internal.common_distributed import MultiProcessTestCase, \
     requires_gloo, requires_nccl, requires_nccl_version, \
     skip_if_not_multigpu, skip_if_lt_x_gpu, get_timeout, skip_if_rocm, \
-    skip_if_rocm_single_process, simple_sparse_reduce_tests, skip_if_win32, \
+    simple_sparse_reduce_tests, skip_if_win32, \
```
Comment (pritamdamania87) [inline test/distributed/test_c10d.py:32]: Are these changes intended as part of this PR? If so, could you update the PR summary to mention why we are removing these skip decorators?

**Comment by pritamdamania87** [comment]
@malfet @seemethere Could you help out in reviewing this PR?

**Comment by malfet** [comment]
@pritamdamania87 this PR is still in draft, isn't it?

**Comment by pritamdamania87** [comment]
> @pritamdamania87 this PR is still in draft, isn't it?

Ah my bad, I guess whenever its ready for review, it would be good to have review from you.

**Comment by gunandrose4u** [comment]
@pritamdamania87 @malfet , this PR is just a PoC to show that we can have a 2GPU Windows test box on Azure which we can run test cases from circleci. It is not a official one for review. I will create a new PR for this scenario. BTW, the Azure credential inside script was removed from Azure subscription every day after I finish E2E test.

**Pull request closed without merging**